src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -425,8 +430,18 @@ <h3 class="panel-title">
               </div>
 
             </div>
-          </div>
 
+            <div class="form-group has-error">
+              <div class="col-sm-offset-3 col-sm-9"
+                   *ngIf="form.hasError('rbdPool')">
+                <br>
+                <span class="help-block"
+                      i18n>It's not possible to create an RBD pool with '/' in the name.</span>
```

Comment:
```suggestion
                      i18n>It's not possible to create a RBD pool with '/' in the name.</span>
```

Reply:
Actually, "an RBD pool" is correct: https://www.ecenglish.com/learnenglish/when-use-a